controllers/dscinitialization/dscinitialization_controller.go

```diff
@@ -200,9 +201,14 @@ func (r *DSCInitializationReconciler) Reconcile(ctx context.Context, req ctrl.Re
 	default:
 		switch platform {
 		case cluster.SelfManagedRhods:
-			err := r.createUserGroup(ctx, instance, "rhods-admins")
-			if err != nil {
-				return reconcile.Result{}, err
+			// Check if user opted for disabling creating user groups
+			if os.Getenv("ODH_USE_EXTERNAL_AUTH") == "true" {
```

Comment:
When upgrading, what is the default value for this env variable?

Reply:
If e value is set in the subscription, it will  persist even later upgrade to new Operator version. Subscription should not get changed.